src/redux/modules/posts.js

```diff
@@ -71,6 +73,13 @@ export const deletePost = id => dispatch => {
     .catch(error => dispatch({ type: DELETE_POST_FAIL, error }));
 };
 
+export const selectPost = id => dispatch => {
+  return dispatch({
```

Comment:
If arrow function does nothing other than return a value, you don't need these: `{}`

You just have to wrap the object you are returning from the function with `()`.

Eg:

```
export const selectPost => id => dispatch => ({ type: SELECT_POST, id})
```